explorer/src/api/constants.ts

```diff
@@ -1,8 +1,16 @@
 export const MIXNODES_API =
   'https://testnet-milhon-explorer.nymtech.net/api/mix-node';
+export const MIXNODE_PING =
+  'https://testnet-milhon-explorer.nymtech.net/api/ping';
 export const GATEWAYS_API =
   'https://testnet-milhon-validator1.nymtech.net/api/v1/gateways';
 export const VALIDATORS_API =
   'https://testnet-milhon-validator1.nymtech.net/validators';
 export const BLOCK_API = 'https://testnet-milhon-validator1.nymtech.net/block';
 export const COUNTRY_DATA_API = 'https://testnet-milhon-explorer.nymtech.net/api/countries';
+export const UPTIME_STORY_API = `https://testnet-milhon-validator1.nymtech.net/api/v1/status/mixnode`; // add ID then '/history' to this.
+export const MIXNODE_STATS_API = `https://testnet-milhon-explorer.nymtech.net/api/mix-node`; // add ID then `/stats` or `/delegations` to this.
+export const DELEGATIONS_API = `https://testnet-milhon-explorer.nymtech.net/api/mix-node`; // add ID then `/stats` or `/delegations` to this.
+
+
+export const MIXNODE_API_ERROR = 'We\'re having trouble finding that record, please try again or Contact Us.'
```

Comment:
Can we break this down a little so that we have something like - 
```
const BASE_URL = "http://testnet-milhon-explorer.nymtech.net"
const BASE_VALIDATOR_URL = "https://testnet-milhon-validator1.nymtech.net"

const COUNTRY_API = `${BASE_API}/block`
const UPTIME_STORY_API = `${BASE_VALIDATOR_URL}/api/v1/status/mixnode
...
```
line 12 and 13 seem to have the same url. Is that intentional?

Reply:
@fmtabbara yes thanks for reminding me, meant to have another look at that.